### ISSUE TITLE
Codegen generating deprecated code for Slick 3.2

### ISSUE THREAD

**Issue by wsargent** [tototoshi/sbt-slick-codegen#10]
From https://github.com/playframework/play-scala-isolated-slick-example/tree/SNAPSHOT -- the codegen package is generating code that is deprecated since 3.2:

```
[warn] /home/wsargent/templates/play-scala-isolated-slick-example/modules/slick/target/scala-2.12/src_managed/main/com/example/user/slick/Tables.scala:10: type JdbcProfile in package driver is deprecated (since 3.2): Use type `slick.jdbc.JdbcProfile` instead of `slick.driver.JdbcProfile`
[warn]   val profile: slick.driver.JdbcProfile
[warn]                             ^
[warn] two warnings found
```

**Comment by tototoshi** [comment]
sbt-slick-codegen depends on slick 3.1 and the generated code is for slick 3.1.
I want to updated the dependency to slick 3.2, but 

 - slick 3.2 has no build against scala 2.10.
 - sbt 0.13 runs only on scala 2.10. 😢 

I'm waiting for sbt 1.0 (or 2.11 supports of sbt 0.13)

**Comment by wsargent** [comment]
Yeah, me too.

**Comment by xuwei-k** [comment]
https://github.com/tototoshi/sbt-slick-codegen/pull/12

**Comment by MasseElch** [comment]
Any progress on this? sbt is 1.0 now, isn't it?

**Comment by tototoshi** [comment]
Released sbt-slick-codegen 1.3.0.
